src/net/smooth/parser.js

```diff
@@ -69,6 +60,29 @@ const profiles = {
   ],
 };
 
+// fourCC values: https://msdn.microsoft.com/en-us/library/ff728116%28v=vs.95%29.aspx
+function extractCodec(fourCC, codecPrivateData) {
```

Comment:
what about having an `extractVideoCodecs` and `extractAudioCodecs` to separate FourCC/codecPD analysis ?